arch/arm64/boot/dts/rockchip/overlays/rk3399-sp2-sc16is752.dts

```diff
@@ -0,0 +1,52 @@
+/dts-v1/;
+/plugin/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/pinctrl/rockchip.h>
+#include <dt-bindings/interrupt-controller/irq.h>
+
+/{
+	metadata {
+		title = "Enable SC16is752 on SPI2";
+		compatible = "rockchip,rk3399";
+		category = "misc";
+		description = "Enable SC16is752 SPI on SPI2.\nINT=38";
+	};
+	
+	compatible = "rockchip,rk3399";
+
+	fragment@0 {
+		target-path = "/";
+		__overlay__ {
+			sc16is752_clk: sc16is752_spi2_0_clk {
+				compatible = "fixed-clock";
+				clock-frequency = <1475600>;
+				#clock-cells = <0>;
+			};
+		};
+	};
+
+	fragment@1 {
+		target = <&spi2>;
+
+		__overlay__ {
+			status = "okay";
+			pinctrl-names = "default";
+			pinctrl-0 = <&spi2_clk &spi2_cs0 &spi2_rx &spi2_tx>;
+			cs-gpios = <&gpio2 RK_PB4 GPIO_ACTIVE_LOW>;
```

Comment:
You already defined cs in pinctrl (which uses SPI hardware CS function), so you don't have to redefine it here using GPIO hardware.